build/.vsts-ci.yml

```diff
@@ -33,1039 +33,1073 @@ resources:
-      testRunTitle: 'XCode'
-    condition: succeededOrFailed()
-  - task: ComponentGovernanceComponentDetection@0
-    displayName: 'Component Detection'
-  - script: rm -rf $(Agent.BuildDirectory)/*
-    displayName: 'Cleanup'
-    condition: always()
-- job: raspberrypi
-  container: raspberrypi-c-buster
-  pool: 
-    name: 'sdk-c--ubuntu-22'
-  displayName: "Raspberry Pi"
-  steps:
-  - script: |
-     $HOME
-     whoami
-     ls -lr
-     pwd
-     chmod +x jenkins/raspberrypi_c_buster.sh
-      ./jenkins/raspberrypi_c_buster.sh
-    displayName: 'Build'
-- job: crosscompile_mips32
-  pool:
-    vmImage: 'ubuntu-20.04'
-  displayName: "Cross Compile (MIPS32)"
-  steps:
-  - script: |
-      sudo apt-get update && sudo apt-get install -y \
-      curl \
-      git \
-      python-software-properties \
-      build-essential \
-      pkg-config \
-      docker.io
-      sudo curl -sL https://deb.nodesource.com/setup_16.x | bash -
-      sudo apt-get install -y nodejs
-    displayName: 'Setup'
-  - script: |
-      cd samples/dockerbuilds/MIPS32
-      docker build --build-arg TARGET_BRANCH=$BUILD_SOURCEBRANCH -t mipsiotbuild:latest . --network=host
-      cd ..
-      docker build -t mipsiotapp:latest . --network=host --file ./MIPS32/Dockerfile_adjunct
-    displayName: 'Build MIPS32'
-- job: crosscompile_arm
-  pool:
-    vmImage: 'ubuntu-20.04'
-  displayName: "Cross Compile (ARM)"
-  steps:
-  - script: |
-      sudo apt-get update && sudo apt-get install -y \
-      curl \
-      git \
-      python-software-properties \
-      build-essential \
-      pkg-config \
-      docker.io
-      sudo curl -sL https://deb.nodesource.com/setup_16.x | bash -
-      sudo apt-get install -y nodejs
-    displayName: 'Setup'
-  - script: |
-      cd samples/dockerbuilds/ARM
-      docker build --build-arg TARGET_BRANCH=$BUILD_SOURCEBRANCH -t armiotbuild:latest . --network=host
-      cd ..
-      docker build -t armiotapp:latest . --network=host --file ./ARM/Dockerfile_adjunct
-    displayName: 'Build ARM'
-- job: DotNET
-  displayName: .NET
-  variables:
-    Codeql.Enabled: $[eq(variables['Build.SourceBranch'], 'refs/heads/main')]
-    Codeql.Language: csharp,python
-  pool: 
-    vmImage: 'windows-latest'
-  steps:
-  - task: CodeQL3000Init@0
-  - task: NuGetCommand@2
-    inputs:
-      command: 'restore'
-      restoreSolution: 'tools\traceabilitytool\traceabilitytool.sln'
-      feedsToUse: 'select'
-      vstsFeed: 'f9b79625-2860-4d92-a4ee-57b03fabfd10/805c1538-e9e5-488f-9e17-6c63b6879ace'
-  - task: DotNetCoreCLI@2
-    inputs:
-      command: 'build'
-      projects: 'tools\traceabilitytool\traceabilitytool.sln'
-  - task: CodeQL3000Finalize@0
-    condition: always()
+    steps:
+    - script: |
+        az login --identity
+        az storage account update -g $STORAGE_ACCOUNT_RESOURCE_GROUP -n $STORAGE_ACCOUNT_NAME --allow-shared-key-access false --default-action Deny
+      displayName: 'Setup Azure Storage Account'
```

Comment:
Should this have a different display name to differentiate it from the previous one? In line 48

Reply:
We could. But in the run it is under a different group, or stage how it's called. So it's not confusing when you see the individual jobs.